button.css

```diff
@@ -0,0 +1,13 @@
+.prevbtn{
```

Comment:
have main class as btn, previous and next can be extension on it eg: btn--previous , btn--next